app/_locales/en/messages.json

```diff
@@ -848,6 +893,21 @@
   "newNetwork": {
     "message": "New Network"
   },
+  "newToMetaMask": {
+    "message": "New to MetaMask?"
+  },
+  "noAlreadyHaveSeed": {
+    "message": "No, I already have a seed phrase"
+  },
+  "protectYourKeys": {
+    "message": "Protect Your Keys!"
+  },
+  "protectYourKeysMessage1": {
+    "message": "Be careful with your seed phrase — there have been reports of websites that attempt to imitate MetaMask. MetaMask will never spontaneously ask for your seed phrase!"
```

Comment:
Nit: should we drop "spontaneously"? Below we have `yourUniqueAccountImageDescription3` plainly saying never.